02-spatial-data.qmd

```diff
@@ -337,7 +338,9 @@ import xarray as xr
 Reading:
 
 ```{python}
-x = xr.open_dataset("data/air.2x2.250.mon.anom.comb.nc")
+url = "https://github.com/geocompr/py/releases/download/0.1/air.2x2.250.mon.anom.comb.nc"
+subprocess.run(["wget", "-nc", url])
```

Comment:
Not sure that will work on Windows. I recommend a pure Python solution like in: https://github.com/anitagraser/ogd-at-lab/blob/main/notebooks/utils/dataaccess.py

Reply:
:+1: